lib/ansible/modules/cloud/docker/docker_network.py

```diff
@@ -394,16 +477,23 @@ def main():
         force=dict(type='bool', default=False),
         appends=dict(type='bool', default=False, aliases=['incremental']),
         ipam_driver=dict(type='str', default=None),
-        ipam_options=dict(type='dict', default={}),
+        ipam_options=dict(type='dict', default={}, removed_in_version='2.12'),
+        ipam_config=dict(type='list', elements='dict', default=None),
+        enable_ipv6=dict(type='bool', default=None),
```

Comment:
You can also leave `default=None` away. That makes it simpler to parse (IMO). (I don't know why it was already in there for some of the existing options...)